src/app/register-item/components/TagInput.tsx

```
@@ -0,0 +1,148 @@
+      e.preventDefault();
+      inputRef.current?.focus();
+    }
+  };
+
+  return (
+    <div>
+      <div
+        className="flex flex-wrap gap-2 p-2 border rounded-md min-h-[40px]"
+        onClick={() => !disabled && inputRef.current?.focus()}
+        onKeyDown={handleContainerKeyDown}
+        role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)